### ISSUE TITLE
Add @type to all properties of the output

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-em-report-tool#221]
Many WCAG-EM Properties do not currently have a @type. This should be resolved as it is important in making the data reusable.
